Developers/Writing_Linter_Bears.rst

```diff
@@ -71,6 +71,17 @@ You can use these parameters to construct the command line arguments. The
 linter expects from you to return an argument sequence here. A tuple is
 preferred. We will do this soon for ``PylintTutorialBear``.
 
+.. note::
+
+    ``create_arguments`` can also be a non-staticmethod. In this case you also
```

Comment:
I wanted to refer to the `@staticmethod` decorator. Though yes you are right, this is no good english :3
I'll rephrase to
> ``create_arguments`` doesn't have to be a static method.